influencetx/core/testing.py

```diff
@@ -10,3 +14,28 @@ class StringContaining(str):
 
     def __eq__(self, other):
         return self in other
+
+
+def assert_mock_logger_called_once(mock_log, log_level, message=None):
+    log_func = getattr(mock_log, log_level)
+    if message:
+        log_func.assert_called_once_with(message)
+    else:
+        log_func.assert_called_once()
+
+
+def create_assert_log_context(log_level):
+    @contextmanager
+    def assert_logged(logger_import, message=None):
+        with mock.patch(logger_import) as mock_log:
+            yield
+            assert_mock_logger_called_once(mock_log, log_level, message=message)
+    return assert_logged
+
+
+def assert_log_warning(logger_import, message=None):
+    return create_assert_log_context('warn')(logger_import, message=None)
+
+
+def assert_log_info(logger_import, message=None):
```

Comment:
I don't see the "assert_log_info" function being used anywhere.  Just FYI, not really a problem.

Reply:
Good catch. Removed. It's easy to add back when needed.